test/integration/iterative_verifier_test.rb

```diff
@@ -1,56 +0,0 @@
-require "test_helper"
-
-class IterativeVerifierTest < GhostferryTestCase
-  def setup
-    seed_simple_database_with_single_table
-  end
-
-  def test_iterative_verifier_succeeds_in_normal_run
-    datawriter = new_source_datawriter
-    ghostferry = new_ghostferry(MINIMAL_GHOSTFERRY, config: { verifier_type: "Iterative" })
-
-    start_datawriter_with_ghostferry(datawriter, ghostferry)
-    stop_datawriter_during_cutover(datawriter, ghostferry)
-
-    verification_ran = false
-    ghostferry.on_status(Ghostferry::Status::VERIFIED) do |*incorrect_tables|
-      verification_ran = true
-      assert_equal 0, incorrect_tables.length
-    end
-
-    ghostferry.run
-    assert verification_ran
-    assert_test_table_is_identical
-  end
-
-  def test_iterative_verifier_fails_if_binlog_streamer_incorrectly_copies_data
```

Comment:
This is simply a ported test from the Go integration test to the ruby integration test when the ruby code was first created, thus why it is redundant.

https://github.com/Shopify/ghostferry/blob/b8147c5ff8653099f6c5e24128aa6fd41ff401e1/test/integration/inline_verifier_test.rb#L115-L138